test/kafka-exactly-once/after-restart.td

```diff
@@ -7,110 +7,98 @@
-{"status":"BEGIN","id":"52","event_count":null,"data_collections":null}
-{"status":"END","id":"52","event_count":{"long": 2},"data_collections":{"array": [{"event_count": 2, "data_collection": "testdrive-input-${testdrive.seed}"}]}}
+  },
+  {
+    "type": "record",
+    "name": "progress",
+    "namespace": "com.materialize.cdc",
+    "fields": [
+      {
+        "name": "lower",
+        "type": {
+          "type": "array",
+          "items": "long"
+        }
+      },
+      {
+        "name": "upper",
+        "type": {
+          "type": "array",
+          "items": "long"
+        }
+      },
+      {
+        "name": "counts",
+        "type": {
+          "type": "array",
+          "items": {
+            "type": "record",
+            "name": "counts",
+            "fields": [
+              {
+                "name": "time",
+                "type": "long"
+              },
+              {
+                "name": "count",
+                "type": "long"
+              }
+            ]
+          }
+        }
+      }
+    ]
+  }
+  ]
 
 $ kafka-ingest format=avro topic=input schema=${schema} timestamp=14
-{"before": null, "after": {"row": {"a": 4, "b": 1}}}
-{"before": null, "after": {"row": {"a": 5, "b": 2}}}
+{"array":[{"data":{"a":4,"b":1},"time":4,"diff":1}]}
+{"array":[{"data":{"a":5,"b":2},"time":4,"diff":1}]}
+{"com.materialize.cdc.progress":{"lower":[4],"upper":[5],"counts":[{"time":4,"count":2}]}}
 
-$ kafka-verify format=avro sink=materialize.public.output_byo sort-messages=true
+$ kafka-verify format=avro sink=materialize.public.output sort-messages=true
 {"before": null, "after": {"row": {"a": 1, "b": 1}}, "transaction": {"id": "1"}}
 {"before": null, "after": {"row": {"a": 1, "b": 2}}, "transaction": {"id": "1"}}
 {"before": null, "after": {"row": {"a": 2, "b": 1}}, "transaction": {"id": "1"}}
 {"before": null, "after": {"row": {"a": 3, "b": 1}}, "transaction": {"id": "1"}}
 
-$ kafka-verify format=avro sink=materialize.public.output_byo sort-messages=true
+$ kafka-verify format=avro sink=materialize.public.output sort-messages=true
 {"before": null, "after": {"row": {"a": 11, "b": 11}}, "transaction": {"id": "2"}}
 {"before": null, "after": {"row": {"a": 22, "b": 11}}, "transaction": {"id": "2"}}
 
-$ kafka-verify format=avro sink=materialize.public.output_byo sort-messages=true
+$ kafka-verify format=avro sink=materialize.public.output sort-messages=true
 {"before": null, "after": {"row": {"a": 3, "b": 4}}, "transaction": {"id": "3"}}
 {"before": null, "after": {"row": {"a": 5, "b": 6}}, "transaction": {"id": "3"}}
 
-$ kafka-verify format=avro sink=materialize.public.output_byo sort-messages=true
+$ kafka-verify format=avro sink=materialize.public.output sort-messages=true
 {"before": null, "after": {"row": {"a": 4, "b": 1}}, "transaction": {"id": "4"}}
 {"before": null, "after": {"row": {"a": 5, "b": 2}}, "transaction": {"id": "4"}}
-
-# can't distinguish "transactions" with real-time timestamping
```

Comment:
Is this a difference here? that the non-byo verification now looks like the old byo verification?  It seems tightened up from the assertions below

Reply:
I removed the non-byo verification because it wasn't a verification of anything. If you see the timestamp had a placeholder (`<TIMESTAMP>`) and we were merely confirming that data in resulted in data out. The test already confirms as much using a CDCv2 stream, no need to also try a realtime timestamped one